src/zk/dist_lock_test.cc

```diff
@@ -43,10 +43,10 @@ void OnLockedCallback() { call_invoked = true; }
 void OnLostCallback() {}
 
 TEST_F(DistLockTest, Lock) {
-    ZkClient client("127.0.0.1:6181", "", 10000, "127.0.0.1:9527", "/rtidb_lock");
+    ZkClient client("127.0.0.1:6181", "", 10000, "127.0.0.1:9527", "/openmldb_lock");
```

Comment:
hi @KikiDotPy , please revert `zk/dist_lock_test.cc` and `zk/zk_client_test.cc`. This task has been done in #2086